atom-client/pom.xml

```diff
@@ -0,0 +1,153 @@
+                    <execution>
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>resources</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>copy-bundles</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${user.home}/.motech/bundles</outputDirectory>
+                            <resources>
+                                <resource>
+                                    <directory>target</directory>
+                                    <includes>
+                                        <include>*.jar</include>
+                                    </includes>
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <configuration>
+                    <instructions>
+                        <Export-Package>
+                            org.motechproject.atomclient.domain;version=${project.version},
+                            org.motechproject.atomclient.repository;version=${project.version},
+                            org.motechproject.atomclient.service;version=${project.version},
+                            org.motechproject.atomclient.service.impl;version=${project.version},
+                            org.motechproject.atomclient.exception;version=${project.version},
+                        </Export-Package>
+                        <Import-Package>
+                            net.sf.cglib.core,
+                            net.sf.cglib.proxy,
+                            net.sf.cglib.reflect,
+                            org.aopalliance.aop,
+                            org.springframework.aop,
+                            org.springframework.aop.framework,
+                            org.springframework.transaction,
+                            org.motechproject.config.service,
+                            org.motechproject.server.config,
+                            org.motechproject.server.config.service,
+                            org.motechproject.mds.service,
+                            com.rometools.fetcher,
+                            org.motechproject.event.listener,
+                            org.motechproject.event.listener.annotations,
+                            *
+                        </Import-Package>
+                        <Bundle-DocURL>
+                            http://modules.motechproject.org/en/latest/atomclient.html
```

Comment:
Doesn't exist (yet). We should create at least a stub doc for this (for example by moving the readme to docs) and a ticket for expansion (if more documention is required)